docs/site/decorators/Decorators_openapi.md

```diff
@@ -191,6 +191,48 @@ You can find specific use cases in
 _The parameter location cookie is not supported yet, see_
 _(https://github.com/strongloop/loopback-next/issues/997)_
 
+### Parameter Decorator to support json objects
+
+{% include note.html content="
+LoopBack has switched the definition of json query params from the `exploded`,
+`deep-object` style to the `url-encoded` style definition in Open API spec.
+" %}
+
+The parameter decorator `@param.query.object` is applied to generate an Open API
+definition for query parameters with JSON values. The generated definition
+currently follows the `url-encoded` style as shown below.
+
+```json
+  {
+    "in": "query"
+    "content": {
+      "application/json": {
+        "schema": {}
+      }
+    }
+  }
+```
+
+The above style where the schema is `wrapped` under content['application/json'],
```

Comment:
nitpick: remove the comma at the end of this line

Reply:
agreed